Gemfile

```diff
@@ -47,6 +47,8 @@ group :development, :test do
   gem "guard-rails"
   gem "guard-rspec"
   gem "json_matchers"
+  gem 'factory_bot_rails'
```

Comment:
This is fine, but it doesn't appear that you actually use the Blobs factory anywhere.  Perhaps this was something that was an idea, but didn't make it into production.  This should have been removed from the PR.

Reply:
I removed it